kotlin-detect-config.yaml

```diff
@@ -148,20 +154,24 @@ style:
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/clusterinteraction/ClusterInteractionFragment.kt"
             - "**/examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/provisioning/EnterNetworkFragment.kt"
             - "**/examples/java-matter-controller/java/src/com/matter/controller/commands/common/MatterCommand.kt"
+            - "**/src/controller/java/generated/java/**/*"
```

Comment:
what unused private properties occur here? Can we avoid?

Reply:
I removed.